### PR TITLE
txs: new queries 

### PR DESCRIPTION
**Pull request by skylarbarrera**
## What changed (plus any additional context for devs)
initial queries

commenting some stuff out cause the parsing the types change and thats a bigger change

### CODE REVIEW

File: src/resources/transactions/consolidatedTransactions.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        queryKey: nftsQueryKey({ address: accountAddress }),
+      });
+ */
+async function parseConsolidatedTransactions(
+  message: TransactionsReceivedMessage,
+  currency: NativeCurrencyKey
+): Promise<RainbowTransaction[]> {
+  const data = message?.payload?.transactions || [];
+
+  return [];
```
Comment (walmat) [inline src/resources/transactions/consolidatedTransactions.ts:137]: Not sure if you wanted to return an empty array here? Also tons of commented out code around here that could be cleaned up if not needed.

File: src/resources/transactions/transaction.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+import { useQuery } from '@tanstack/react-query';
+import {
+  consolidatedTransactionsQueryFunction,
+  consolidatedTransactionsQueryKey,
+} from './consolidatedTransactions';
+import { useAccountSettings } from '@/hooks';
+import { RainbowNetworks, getNetworkObj } from '@/networks';
+import { rainbowFetch } from '@/rainbow-fetch';
+import { ADDYS_API_KEY } from 'react-native-dotenv';
+//import { parseTransaction } from '@/parsers/transactions';
```
Comment (walmat) [inline src/resources/transactions/transaction.ts:17]:
```suggestion
```

File: src/resources/transactions/transaction.ts [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        currency: currency.toLowerCase(),
+      },
+      timeout: 20000,
+      headers: {
+        Authorization: `Bearer ${ADDYS_API_KEY}`,
+      },
+    });
+    const tx = response?.data?.payload?.transaction || [];
+
+    return undefined;
```
Comment (walmat) [inline src/resources/transactions/transaction.ts:68]: Is this intentional?

**Review by walmat — COMMENTED** [review]
Looks good a couple small comments

File: src/resources/transactions/transaction.ts (hunk 3 above)
Comment (skylarbarrera) [reply src/resources/transactions/transaction.ts:68]: this is just to fix lint, so we can get this in as a smaller file change

File: src/resources/transactions/transaction.ts (hunk 2 above)
Comment (skylarbarrera) [reply src/resources/transactions/transaction.ts:17]: should be nit, gonna uncomment that right after, just trying to reduce # of files for review at a time

File: src/resources/transactions/consolidatedTransactions.ts [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+// ///////////////////////////////////////////////
+// Query Key
+
+export const consolidatedTransactionsQueryKey = ({
+  address,
+  currency,
+  chainIds,
+}: ConsolidatedTransactionsArgs) =>
+  createQueryKey(
+    'consolidatedTransactionss',
```
Comment (benisgold) [inline src/resources/transactions/consolidatedTransactions.ts:38]: id has an extra "s" at the end

**Review by benisgold — COMMENTED** [review]
the tiniest of nits

File: src/resources/transactions/consolidatedTransactions.ts (hunk 4 above)
Comment (skylarbarrera) [reply src/resources/transactions/consolidatedTransactions.ts:38]: fuck!

**Review by benisgold — APPROVED** [review]

**Review by walmat — APPROVED** [review]

**Pull request merged**